alf/algorithms/rl_algorithm.py

```diff
@@ -371,11 +371,19 @@ def summarize_reward(self, name, rewards):
                         torch.mean(r),
                         average_over_summary_interval=True)
 
-    def summarize_rollout(self, experience):
+    @alf.configurable(whitelist=["inject_summary"])
+    def summarize_rollout(
+            self,
+            experience: Experience,
+            inject_summary: Optional[Callable[[Experience], None]] = None):
```

Comment:
suggest changing name from `inject_summary` to `custom_summary`.

Reply:
Sounds good. Fixed.